generators/generator-adaptive-bot/generators/app/templates/assets/NuGet.config

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <packageSources>
-    <add key="botbuilder-v4-dotnet-daily" value="https://botbuilder.myget.org/F/botbuilder-v4-dotnet-daily/api/v3/index.json" protocolVersion="3" />
-  </packageSources>
-</configuration>
+	<packageSources>
+		<clear/>
+		<add key="BotBuilder.myget.org" value="https://botbuilder.myget.org/F/botbuilder-v4-dotnet-daily/api/v3/index.json" protocolVersion="3" />
+	</packageSources>
```

Comment:
Two things:

1) Can we fix indentation?
2) If I understand the documentation correctly, adding <clear /> causes other package sources to be ignored, so we'd only look at botbuilder. Is that correct? If so, is this intentional? I don't think we should necessarily only look at that source - it should be in addition to any other preconfigured sources.

Reply:
1. Yes
2. That's correct and intentional! Going forward the open source policy requires us to only use a single feed and add additional feeds as upstream sources to said feed. For now we must use the botbuilder myget which includes nuget.org as an upstream source, but are looking into other options for all the botbuilder repos. See more at https://aka.ms/nugetmultifeed

Reply:
Thank you for clarifying on the NuGet.config. The requirement makes sense. My concern with this particular file though is that this is included in the generated output from yeoman. I wouldn't put that same enforcement on this particular one.

Reply:
That's fair, tracking at #567. If I revert this to multifeed now the builds across the repository will break. We can dig deeper and I believe there's a process to get whitelisted.

Reply:
OK, let's talk about this at standup. Might be misunderstanding something here. Thank you for creating an issue to track!